File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars;
using DevExpress.Utils.Drawing.Animation;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        AutoHideBar bar1;
        AutoHideBar bar2;
        public Form1()
        {
            InitializeComponent();
            bar1 = new AutoHideBar(barManager1, BarDockStyle.Top);
            bar2 =  new AutoHideBar(barManager1, BarDockStyle.Bottom);
            for (int i = 0; i < 10; i++)
            {
                bar1.AddItem(new BarButtonItem(barManager1, "Test"));
                bar2.AddItem(new BarButtonItem(barManager1, "Test"));
            }
         
        }

        private void Form1_Shown(object sender, EventArgs e)
        {
            bar1.HideBar();
            bar2.HideBar();
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars
Imports DevExpress.Utils.Drawing.Animation

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form
		Private bar1 As AutoHideBar
		Private bar2 As AutoHideBar
		Public Sub New()
			InitializeComponent()
			bar1 = New AutoHideBar(barManager1, BarDockStyle.Top)
			bar2 = New AutoHideBar(barManager1, BarDockStyle.Bottom)
			For i As Integer = 0 To 9
				bar1.AddItem(New BarButtonItem(barManager1, "Test"))
				bar2.AddItem(New BarButtonItem(barManager1, "Test"))
			Next i

		End Sub

		Private Sub Form1_Shown(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Shown
			bar1.HideBar()
			bar2.HideBar()
		End Sub
	End Class
End Namespace